### DISCUSSION TITLE
cant connect to AP

### DISCUSSION THREAD

**Discussion by PapaSchIumpf** [RaspAP/raspap-webgui#1596] [Q&A] [answered]
Hi everyone,

i use raspap with my rpi-5 with the the newest raspberry pi OS full version and got issues with the acess point.

The problem is that i really often cant get a connection with the AP.

There are several devices connected to the pi over the AP, like shelly plugs and for most it works fine, they connect to it and are avaible in the cloud, because i have ethernet connected.

Also when i restart the devices or the pi, they connect automatically again, so that works fine.

But i cant add new devices or just sometimes after restarting the hotspot several times.

I can reach the RaspAp UI over the ethernet connection, that works, also the AP is visible.

I dont understand and i have not lot experience regarding network issues, so what could be the reason?

debug file:
[raspap_debug(3).log](https://github.com/user-attachments/files/15846522/raspap_debug.3.log)

i have attached my debug log, maybe that helps.

Thanks in advance.

**Comment by PapaSchIumpf** [comment]
i found out that it has to do with my esp32 devices that i also have connected to the pi...

when i displug them i can normally connect to raspap over the AP but when my esp32 devices connect and i try to connect, no chance

when i am connectet over the AP and connect my esp32 device after that, no problem, till i disconnect and try to reconnect to the AP again

the esp itself gets an ip adress and works fine, only issue is the not connectable AP

**Reply by Noschvie** [reply]
Which firmware are you running with the ESP32?

**Comment by PapaSchIumpf** [comment]
i have written my own code, i use the esp32 as a asyncwebserver with a webinterface
There are no issues when i connect them to my"normal" router, but with raspap...

I have checked the code of my esp several times, but cant find an issue
i use espasyncwebserver library for the webserver

as soon as the esp is connected to raspap, no other device can connect
when i plug the esp off its possible

really dont understand......

thanks for your help

**Reply by Noschvie** [reply]
Is this reproducible running the ESP32 with Tasmota ?

**Reply by PapaSchIumpf** [reply]
i tried it now, its interesting, with tasmota flashed on my device i cant connect at all to raspap.....
when i choose my "normal" router it works, but when i try to connect to raspap i get this

00:00:22.000 WIF: Connecting to AP1 sprosskraft-rpi5-AP in mode HT20 as tasmota-CBAF60-3936...
00:00:33.109 WIF: Connect failed with AP timeout

but with my other devices no problem, i can connect to raspap with no problem

**Comment by PapaSchIumpf** [comment]
now i found that the same happens when i connect my phone to raspap, still no solution

**Comment by PapaSchIumpf** [comment]
it seems like i have a maximum number of clients, i cant connect more than 6 devices...
i tried changing the value in the settings for that but nothing changed

**Reply by billz** [reply]
hostapd, the service administered by RaspAP that provides the AP, is capable of supporting more than this number of clients. My guess is you're not running into this constraint. See this FAQ for an explanation https://docs.raspap.com/faq/#maxclients

If your AP is operating on the 5 GHz band, be sure that your client devices have compatible WLAN chipsets (all new phones support WiFi 5 and many support WiFi 6, but users try all sorts of older hardware). Also, if you haven't done so already, enable logging for both the hotspot and DHCP server, restart the services, then try connecting a device (clearing the log before doing so can be helpful). Log entries will capture clients' attempts to associate with the AP and indicate any problems. For troubleshooting tips, see this FAQ https://docs.raspap.com/faq/#noip

Hope this is useful.

**Reply by PapaSchIumpf** [reply]
thank you, but i think this still might be the issue...
i have checked your link and found this too:
https://github.com/RaspAP/raspap-webgui/issues/462#issuecomment-588367233
there are other users who also can only connect 7 or 8 clients to the AP, same like me
i can connect 7 in total, i tried different devices out, its always the 7 clients total, the device doesnt matter...

**Reply by billz** [reply, marked as answer]
As discussed in this thread https://github.com/raspberrypi/linux/issues/3010#issuecomment-748450967 the solution for many users is to use an external wireless adapter.
